packages/expo-image-picker/ios/MediaHandler.swift

```diff
@@ -104,7 +104,9 @@ internal struct MediaHandler {
     guard let fileSystem = self.fileSystem else {
       throw FileSystemModuleNotFoundException()
     }
-    let directory = fileSystem.cachesDirectory.appending("ImagePicker")
+    let directory =  fileSystem.cachesDirectory.appending(
+      fileSystem.cachesDirectory.hasSuffix("/") ? "" : "/" + "ImagePicker"
```

Comment:
I assume this was tested in bare context and the leading `/` wasn't needed, but if I'm mistaken about that then we can just replace this ternary with `"/ImagePicker"`